package/src/scripts/helpers/parseComponent.ts

```diff
@@ -3,8 +3,9 @@ import getComponent from '@scripts/helpers/getComponent';
 
 export const DOMParser = (self: VanillaCalendar, template: string) => (
 	template.replace(/[\n\t]/g, '').replace(/<#(?!\/?Multiple)(.*?)>/g, (_, p1) => {
-		const component = getComponent(p1.replace(/[/\s\n\t]/g, ''));
-		const html = component ? component(self) : '';
+		const [__, comp, attribute] = /(.*)\s?\[(.*)\].*/g.exec(p1) || [];
```

Comment:
I'm sure this regex could be improved and perhaps merged with the regex just above on the previous line but I played it safe and couldn't figure out the best regex to use, feel free to change. The regex is to extract any optional attribute that I added to some of the component like `<#ArrowPrev [month] />` or `<#ArrowPrev [year] />`